src/support/cleanse.h

```diff
@@ -8,6 +8,8 @@
 
 #include <stdlib.h>
 
+// Securely erase a span of memory in a way guaranteed not to be
```

Comment:
To be fair in this case since OPENSSL_cleanse() exists in an external shared object, we know it won't be optimized out.   Should I back out this comment though?